protocol/app/upgrades/v5.0.0/upgrade.go

```diff
@@ -204,6 +206,39 @@ func initializeOIMFCaps(
 	}
 }
 
+func voteExtensionsUpgrade(
+	ctx sdk.Context,
+	keeper consensusparamkeeper.Keeper,
+) {
+	currentParams, err := keeper.Params(ctx, &consensustypes.QueryParamsRequest{})
+	if err != nil {
+		panic(fmt.Sprintf("failed to retrieve existing consensus params in VE upgrade handler: %s", err))
+	}
+	if currentParams.Params.Abci.VoteExtensionsEnableHeight != 0 {
+		panic(fmt.Sprintf(
+			"unable to update VE Enable Height since its current value of %d is already non-zero",
+			currentParams.Params.Abci.VoteExtensionsEnableHeight))
+	}
+	veEnableHeight := ctx.BlockHeight() + 4
+	currentParams.Params.Abci.VoteExtensionsEnableHeight = veEnableHeight
```

Comment:
Nit: inline and declare constant in `v5.0.0/constants.go` for readability

`currentParams.Params.Abci.VoteExtensionsEnableHeight = ctx.BlockHeight() + VoteExtensionEnableHeightDelta`